rosdep/python.yaml

```
@@ -8959,6 +8959,10 @@ python3-pytest-mock:
   rhel: ['python%{python3_pkgversion}-pytest-mock']
   ubuntu: [python3-pytest-mock]
 python3-pytest-timeout:
+  alpine:
+    '*': null
+    '3.14': [py3-pytest-timeout]
+    '3.17': [py3-pytest-timeout]
```

If it's supposed to be supported in the next Alpine base version,
```suggestion
    '3.11': null
    '3.8': null
    '*': [py3-pytest-timeout]
```
would be better